src/Abc.Zebus/Transport/DefaultZmqOutboundSocketErrorHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Abc.Zebus.Transport
+{
+    public class DefaultZmqOutboundSocketErrorHandler : IZmqOutboundSocketErrorHandler
+    {
+        public void OnConnectException(PeerId peerId, string endPoint, Exception exception)
+        {
+        }
+
+        public void OnDisconnectException(PeerId peerId, string endPoint, Exception exception)
+        {
+        }
+
+        public void OnSendFailed(PeerId peerId, string endPoint, MessageTypeId messageTypeId, MessageId id)
+        {
+        }
+    }
+}
```

Comment:
Wouldn't it be better to move the log messages to this handler?

Reply:
I wanted to keep the log messages even if a custom `IZmqOutboundSocketErrorHandler` is implemented.

Reply:
ok, fair enough